sky/backends/backend_utils.py

```diff
@@ -680,6 +684,34 @@ def remove_cluster(
             if 'Host' in config_str:
                 break
 
+    @classmethod
+    @timeline.FileLockEvent(ssh_conf_lock_path)
+    def remove_cluster(
+        cls,
+        cluster_name: str,
+        ip: str,
+        auth_config: Dict[str, str],
+        docker_user: Optional[str] = None,
+    ):
+        """Remove authentication information for cluster from local SSH config file
```

Comment:
Style nit: keep it single line for the first sentence of the docstr.

Reply:
I think yapf throws an error if it is too long. Do you have an idea on what to do in this case?